File: WebAPP/src/app/models/PhieuBaoDuongDto.ts
export class PhieuBaoDuongDto {
  id: number;
  ma: string;
  nhanVienId: number;
  trangThai: number | null;
  createTime: Date | null;
  tongThietBi: number | null;
  danhSachThietBi: number[];
}

File: WebAPI/WebAPI/Models/PhieuBaoDuong/PhieuBaoDuongDto.cs
using System.ComponentModel.DataAnnotations;

namespace WebAPI.Models.PhieuBaoDuong
{
    public class PhieuBaoDuongDto
    {
        public int Id { get; set; }
        public string Ma { get; set; }
        public int? NhanVienId { get; set; }
        public int? TrangThai { get; set; }
        public DateTime? CreateTime { get; set; }
        public int? TongThietBi { get; set; }
        public List<int> DanhSachThietBi { get; set; }
    }
}
